### PR TITLE
Feat: rebrand web wallet to flash

### LINKED ISSUE #1: [UI] Rebrand Web Wallet to Flash - Bounty #1
## Task:
Rebrand entire web wallet to be branded and themed for Flash. 

- [ ] Replace all blink images with flash images
- [ ] Replace all galoy text (Blink, BBW, Bitcoin Beach Wallet, Galoy, etc) to Flash
- [ ] Modify color scheme to match flash color scheme 

Bounty details posted [HERE](https://community.sphinx.chat/tickets)

### PR DESCRIPTION
**Pull request by YLeight**
Resolves: #1

### CODE REVIEW

**Comment by islandbitcoin** [comment]
colors look good -  can we change the footer  to "Powered by Flash" with the URL being https://getflash.io

File: src/components/__snapshots__/root.spec.tsx.snap [hunk 1]
```diff
@@ -497,11 +497,11 @@ exports[`Root appRoutes renders Home and matches snapshot 1`] = `
       >
         Powered By 
         <a
-          href="https://galoy.io/"
+          href="https://flash.io/"
```
Comment (islandbitcoin) [inline src/components/__snapshots__/root.spec.tsx.snap]: getflash.io

**Review by islandbitcoin — CHANGES_REQUESTED** [review]
Small change needed then LGTM

File: src/components/__snapshots__/root.spec.tsx.snap (hunk 1 above)
Comment (YLeight) [reply src/components/__snapshots__/root.spec.tsx.snap]: Done

**Review by islandbitcoin — APPROVED** [review]
LTGM

**Pull request merged**